src/main/java/com/google/idea/perf/TracerConfig.kt

```diff
@@ -43,6 +43,13 @@ object TracerConfig {
         lock.withLock {
             val classConfig = classConfigs.getOrPut(classJvmName) { ClassConfig() }
             classConfig.methodNames.add(methodName)
+
+            for ((signature, methodId) in classConfig.methodIds) {
```

Comment:
Let's add a short comment above this line, something like "Re-enable tracepoints that were disabled via untrace."

Reply:
Sure